### PR TITLE
chat work

### PR DESCRIPTION
**Pull request by gibbonsdav**
I made a front end component(Chat.js) and tried to link it to the back end. Im not sure how to make it work from here. I think it needs a network to work?

### CODE REVIEW

File: client/src/components/Chat.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from "react"
+import io from "socket.io-client"
+
+export default props => {
+  const [text, setText] = useState([])
```
Comment (girafffene) [inline client/src/components/Chat.js]: why are these arrays and not strings? just asking --curious

File: client/src/components/Chat.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  const [messages, setMessages] = useState([])
+
+  const socket = io.connect("http://") //???//
```
Comment (girafffene) [inline client/src/components/Chat.js]: this can remain as localhost:3000

File: client/src/components/Chat.js (hunk 1 above)
Comment (gibbonsdav) [reply client/src/components/Chat.js]: Dont know, I was just trying to emulate what Mike was doing on the initial socket lecture. Could be wrong though.

Comment (girafffene) [reply client/src/components/Chat.js]: we'll keep it like that then

**Pull request merged**